lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -25,8 +25,10 @@ public function register_routes() {
 					'context'          => array(
 						'default'      => 'view',
 					),
-					'type'            => array(),
```

Comment:
Removed `type` as it's not used in the controller at all - this was from before the posts controller abstraction.